### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#794] [Galaxy Trivia] [answered]
🔺⬛⬜🟩
⬜⬇️🟥🟩
⬜🟦👑

**Comment by SANKARAMDAS** [comment]
🔺⬛⬜🟩 flag of the United Arab Emirates 🇦🇪
 ⬜⬇️🟥🟩 The national flag of Azerbaijan 🇦🇿
⬜🟦👑 The flag of San Marino 🇸🇲

**Comment by peterbaccaro** [comment, marked as answer]
1. PALESTINE
2. MADEGASCAR
3. SAN MARINO

![image](https://github.com/ParanoidUser/yolo/assets/74666079/61a16d6f-095b-4b4d-8a81-3f980891b13f)
